recipes/1-Insights/conditioned-insights.ipynb

```diff
@@ -0,0 +1,309 @@
+    "sex_1_accuracy = t.get_prediction_stats(action_condition={'sex': 1})['target']['accuracy'].round(2)\n",
+    "print(f\"Sex 0 accuracy: {sex_0_accuracy}\")\n",
+    "print(f\"Sex 1 accuracy: {sex_1_accuracy}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we see a difference in the model performance between males and females, this may indicate that there may be further bias that needs to be investigated and/or this Trainee is not suitable for inference on both sexes.\n",
+    "\n",
+    "If this Trainee and data were used to make decisions on loan acceptance based on income, these differences in performance may indicate that this Trainee and data may be suitable for predictions on males but not females. This can lead to actions like gathering better data that captures the characteristics of females better."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Action Condition and Context Condition\n",
+    "\n",
+    "Another way to see how different groups can differ is by specifying both action and context conditions. Unlike when only the action condition is specified, specifying both conditions will hold out `ALL` of the action set. After a case from the action set is used to make a prediction, unlike when just the action condition is specified, that case will continue to be held out. Thus, using this method, the cases from the context set and action set remain separate during the entire process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Sex 1 accuracy: 0.71\n",
+      "Sex 0 accuracy: 0.89\n"
+     ]
+    }
+   ],
+   "source": [
+    "sex_1_conditioned_accuracy = t.get_prediction_stats(context_condition={'sex': 0}, action_condition={'sex': 1})['target']['accuracy'].round(2)\n",
+    "sex_0_conditioned_accuracy = t.get_prediction_stats(context_condition={'sex': 1}, action_condition={'sex': 0})['target']['accuracy'].round(2)\n",
+    "print(f\"Sex 1 accuracy: {sex_1_conditioned_accuracy}\")\n",
+    "print(f\"Sex 0 accuracy: {sex_0_conditioned_accuracy}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This method may reveal extra insights that conditioning on the action set may not since the entire action set is being held out, the prediction stats are based purely on the context set. For example, if we see that when only cases that are males are used to predict cases that are females, and the performance drops greatly from the global predicition stats, then this provides further indication in the difference in the underlying data and relationships among females vs males. A difference in the performance when the holdouts are reversed may indicate that the data overall suits one segment vs another. "
```

Comment:
"this method may reveal extra insights that conditioning on the action set may miss"

Reply:
Can you split the sentence that starts "For example..." in two? That will help with understandability.